websocketOutportDriver/payloadParser.go

```diff
@@ -150,3 +136,22 @@ func prepareArgsSaveBlock(args *outport.ArgsSaveBlockData) *outport.ArgsSaveBloc
 		IsImportDB:             args.IsImportDB,
 	}
 }
+
+func prepareLogs(initial []*dataCore.LogData) []*dataCore.LogData {
+	res := make([]*dataCore.LogData, 0, len(initial))
+	for _, logHandler := range initial {
+		res = append(res, &dataCore.LogData{
+			LogHandler: logHandler.LogHandler,
+			TxHash:     hex.EncodeToString([]byte(logHandler.TxHash)),
+		})
+	}
+	return res
+}
+
+func prepareTxs(initial map[string]dataCore.TransactionHandlerWithGasUsedAndFee) map[string]dataCore.TransactionHandlerWithGasUsedAndFee {
```

Comment:
a better naming would be `setKeysAsHexEncoded`

Reply:
I prefer this name because maybe in the future will add more modifications here not only  the `hex-encoding` part of the hashes.